packages/chat/commands/cover-traffic.ts

```diff
@@ -0,0 +1,73 @@
+import { encode, decode } from 'rlp'
+import type HoprCoreConnector from '@hoprnet/hopr-core-connector-interface'
+import type Hopr from '@hoprnet/hopr-core'
+import { AbstractCommand } from './abstractCommand'
+
+const INTERVAL = 1000
+
+export class CoverTraffic extends AbstractCommand {
+  private seq: number = 0
+  private timeout: NodeJS.Timeout | undefined
+  private registered: boolean
+
+  private messagesSent: number
+  private messagesReceived: number
+  private totalLatency: number
+
+  private identifier: number
+  constructor(public node: Hopr<HoprCoreConnector>) {
+    super()
+    this.messagesSent = 0
+    this.messagesReceived = 0
+    this.totalLatency = 0
+    this.identifier = Math.random()
+  }
+
+  public name() {
+    return 'cover-traffic'
+  }
+
+  public help() {
+    return 'Generate chaff messages to provide cover (start/stop)'
+  }
+
+  private tick() {
+    const payload = encode([this.identifier, this.seq++, Date.now()])
+    this.node.sendMessage(payload, this.node.getId())
+    this.messagesSent++
+    this.timeout = setTimeout(this.tick.bind(this), INTERVAL)
```

Comment:
would add a small comment:
```// calls itself every```

Reply:
can you elaborate what is unclear about this? Happy to document, but self calling timers are fairly common? Would like to know what is strange?

Reply:
am fine with it calling itself, it's just usually functions don't do that so comment on L38 would be nice for the first-time readers

Reply:
Maybe I'm showing my age with `.bind` - would it still need a comment if I did `setTimeout(() => this.tick(), INTERVAL)`

Reply:
haha, dunno if `.bind` is too old, but yeah I would still add a comment, this is me nitpicking though so feel free to ignore